README.md

```diff
@@ -1,2 +1,24 @@
-# substance_presenter
+SubstancePresenter
+==================
+
 An easy to use presenter implementation
+
+
+[![Build Status](https://travis-ci.org/substancelab/substance_presenter.svg?branch=master)](https://travis-ci.org/substancelab/substance_presenter)
+
+
+Usage examples
+--------------
+
+  # Presenting a single model:
+  presented_model = Substance::Presenter.present(model, view_context)
```

Comment:
You should probably format this as code, ie indent by 4 spaces or add a code block around it.